client/src/containers/CampaignAdmin/tests/CampaignAdminList-test.js

```diff
@@ -8,6 +8,7 @@ import React from 'react';
 import { Component as CampaignAdminList } from '../CampaignAdminList';
 import Enzyme, { shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import { setImmediate } from 'core-js';
```

Comment:
https://stackoverflow.com/a/68724767/10936596
Fixes `ReferenceError: setImmediate is not defined` which is test-specific.